ui/lib/apps/SearchLogs/components/SearchProgress.tsx

```diff
@@ -42,10 +43,11 @@ function leafNodeProps(state: number | undefined) {
 function renderLeafNodes(tasks: LogsearchTaskModel[]) {
   return tasks.map((task) => {
     const title = task.target?.display_name ?? ''
+    const size = getValueFormat('bytes')(task.size!, 1)
```

Comment:
If `task.size === 0` here, means the task is still running or aborted.
We should not display `0B` here.

Reply:
Is it OK like this? 

```ts
function renderLeafNodes(tasks: LogsearchTaskModel[]) {
  return tasks.map((task) => {
    let title = task.target?.display_name ?? ''
    if (task.size) {
     title += ' ' + getValueFormat('bytes')(task.size!, 1)
    }
    return (
      <TreeNode
        key={`${task.id}`}
        title={title}
        {...leafNodeProps(task.state)}
      />
    )
  })
}
```

Reply:
OK